### PR TITLE
add section about situation CoC report involves core team member

### PR DESCRIPTION
**Pull request by fmichonneau**
The @carpentries/executive-council has asked the Executive Team to contact Community Initiatives to understand how a report of a CoC incident involving a member of the Core Team would need to be handled. After consultation with Community Initiatives and on Lex's suggestion, we are adding the recommendations to the handbook.

I think this is the most appropriate place for this but let me know if it should go somewhere else.

Could members of the @carpentries/code-of-conduct-committee review these changes?

### CODE REVIEW

File: topic_folders/policies/incident-response.md
Comment (ErinBecker) [inline topic_folders/policies/incident-response.md:50]:
```suggestion
The process below applies to any CoC incidents involving [Core Team](https://carpentries.org/team/) members

* If there are CoC reports either about or from (or in any way involving) Core
```

File: topic_folders/policies/incident-response.md
Comment (ErinBecker) [inline topic_folders/policies/incident-response.md:54]:
```suggestion
* Non-U.S. based Core Team members are employees of Velocity Global. If a Velocity Global employee is involved, we will
```

File: topic_folders/policies/incident-response.md
Comment (ErinBecker) [inline topic_folders/policies/incident-response.md:55]:
```suggestion
  need to bring Velocity Global for country-specific advice. Dave McGee at CI is our liaison with Velocity Global ([david@communityin.org](mailto:david@communityin.org)).
```

File: topic_folders/policies/incident-response.md
Comment (ErinBecker) [inline topic_folders/policies/incident-response.md:56]:
```suggestion
* In cases involving CI or Velocity Global employees that are definitely not harassment, it is ok
```

File: topic_folders/policies/incident-response.md
Comment (ErinBecker) [inline topic_folders/policies/incident-response.md:64]:
```suggestion
CoCc (currently François Michonneau ([francois@carpentries.org](mailto:francois@carpentries.org)) . The Core Team CoCc liaison will elevate anything that comes in by or about
```

File: topic_folders/policies/incident-response.md
Comment (ErinBecker) [inline topic_folders/policies/incident-response.md:66]:
```suggestion
Initiatives or the Velocity Global if needed. If a report involves the Core Team CoCc
```

**Comment by karinlag** [comment]
I have read through it, and first of all, I think this text would rather belong in topic_folders/policies/enforcement-guidelines.html.

Second, this issue needs to be further aligned with our other processes, for instance how we would treat a CoC report where the reported person was an Executive Council member.

Third, I am not very comfortable about the directive to effectively break confidentiality in this way. Not without further discussions regarding what this will do to the chances of people actually reporting issues. 

Fourth, I am not comfortable about the language in the text. We have conciously avoided using terms such as harassment in our other documents due to it being a very loaded word, and also because people who might have issues to report might not feel that what they have experienced consitutes harassment, altough it might actually be. Also, I'd like to add that the CoCc does not know what the CI would deem to be harassment, and thus it would be difficult for us to judge what should and should not be reported onwards. 

Fifth, I take issues with deeming things such as these as not being harassment: 
" Examples of types of things that
  might be reported to the CoCc but that are not harassment are a workshop
  participant swearing at an Instructor, or a community member who is rude and
  pushy but not in any way related to sexual advances or protected categories."

All of these break with our current standards of behavior, and we have at previous times censured people for this kind of behavior. Listing them in ways that indicate that they are not harassment can make such issues seem less serious, and might lead to people not reporting them to us. 

Last but not least, the text needs to be reworked for understandability for a non-corporate/non-US audience. This is a publicly facing document that people who might consider reporting things are likely to read. This text uses several terms that might not be immediately understood by our global audience.  Also it is clear that this is written from inside of a US-corporate context, where employees probably have a corporate rule set that they need to comply with. If this is the case, this situation has to be clearly explained in the text so that people understand how this fits in with the rest of things. 

If I may, I would suggest that this text takes a swing by the CoCc, and we can work on aligning it with our other processes, and then swing it by the staff and the EC to make sure we get a good process on it.

File: topic_folders/policies/incident-response.md
Comment (lexnederbragt) [inline topic_folders/policies/incident-response.md:57]: Not sure adding two small examples here helps or rather confuses the person that needs to decide whether something is harassment or not.

**Comment by malvikasharan** [comment]
Pinging this thread to see if we should make some decision around this sooner:

CoC enforcement for a community member vs employee may look different. I have come up with some solution that includes the following section in the enforcement ([see the draft on my repo](https://github.com/alan-turing-institute/open-community-building/blob/88e6082abc19a3601a78cca2836f99d9c2c246bc/CODE_OF_CONDUCT.md)):

> 15.2 If the breach is considered outside the scope of this Code of Conduct
>
> 15.2.1 If the reportee is an employee of the [org name]
>
> Code of Conduct should be considered as a norm-setting mechanism for the Open Research Community Building Team before any involvement of HR in the interactions that take place within the team.
>
> If the reportee is a [org name] employee, for instance, Community Managers and senior researchers, and the incident is considered out of the scope of this Code of Conduct due to the severity of the matter/case, an HR process (formal or informal), will be started. In this case, the report will be passed to the HR and/or appropriate personnel under the Turing policies and their policy related enforcement mechanism will take precedent over this Code of Conduct enforcement.
>
> All the employees have the obligation to read and understand the staff policies, which include:
> [insert link to list of HR policies]

**Comment by kariljordan** [comment]
This is extremely helpful, and would be a perfect topic to discuss at the upcoming Executive Council meeting @malvikasharan. Should I put this issue on the agenda?

File: topic_folders/policies/incident-response.md
Comment (kariljordan) [reply topic_folders/policies/incident-response.md:64]: @ErinBecker this needs to be updated to say that I am the CT liaison.

File: topic_folders/policies/incident-response.md
Comment (yochannah) [reply topic_folders/policies/incident-response.md:53]: can I suggest that an attached document with individuals listed as designated contacts might be clearer than having to update the CoC every time a staff member moves?

**Review by yochannah — APPROVED** [review]
Adding a review here - this looks sensible to me, with my only suggestion being that names should be in a separate file that gets updated when roles change, rather than here specifically

**Comment by malvikasharan** [comment]
@yochannah Do you mean we need a link to the Core team member by adding this link: https://carpentries.org/team/) or the link to CI-HRs: https://communityin.org/meet-us/team/hr/? Or Both?
@kariljordan would that be ok? I can edit that in quickly and then this PR will be ready to merge in my opinion.

**Comment by kariljordan** [comment]
@malvikasharan it is perfectly fine to link either of those pages. It looks like you're waiting for clarity from @yochannah about whether to link the CI HR page or the Core Team staff page.

File: topic_folders/policies/incident-response.md
Comment (malvikasharan) [reply topic_folders/policies/incident-response.md:53]: Agreed! Some comments are in the PR thread.

**Comment by ErinBecker** [comment]
@malvikasharan - thanks for pinging me. I see your comments in the text thread, but don't see where those have been integrated into the PR (the most recent changes are from October 2021). Your comment above says that the PR would be ready to merge after edits from you, so I'm wondering if your edits got lost. If I'm misunderstanding, please let me know and I will edit to reflect my understanding of your comments.

**Comment by maneesha** [comment]
Assigning to @acrall  as our new CoC liaison.

**Comment by kariljordan** [comment]
Hi @maneesha, I am still the official liaison while Alycia gets onboarded. We anticipate her taking over in Q2. Would you like me to review and merge this PR?

**Comment by malvikasharan** [comment]
@ErinBecker and @kariljordan I don't have access to edit this file, so if we can merge this PR,  can create a new one suggesting my edits.

**Comment by maneesha** [comment]
@kariljordan  yes, if things look good to you, please merge.  Thank you.

**Review by kariljordan — APPROVED** [review]

**Pull request merged**
